features/accounts/edit/profile-form/profile-form.tsx

```diff
@@ -64,77 +62,63 @@ export const ProfileForm = () => {
   return (
     <div className={style.formContainer}>
       <form className={style.form} onSubmit={handleSubmit(onSubmit)}>
+        {process.env.NEXT_PUBLIC_MODE === 'development' && <DevTool control={control} />}
         <Controller
           control={control}
-          name={'username'}
+          name={'userName'}
           render={({ field, fieldState }) => (
             <TextField
-              {...field}
-              errors={fieldState?.error?.message}
-              inputtype={'text'}
-              label={'Username'}
-              onChange={field.onChange}
-              onFocus={() => clearErrors('username')}
```

Comment:
In my opinion, implicitly removing the user input error at focus is counterintuitive. The user should type the correct value themselves and then see the error disappear. Auto-disappearing an error without any user interaction can be confusing and be perceived as a bug.